dev/PushNotifications/PushNotificationManager.h

```diff
@@ -6,16 +6,21 @@
 
 namespace winrt::Microsoft::Windows::PushNotifications::implementation
 {
+    static winrt::Windows::ApplicationModel::Background::IBackgroundTaskRegistration s_taskRegistration = nullptr;
+    static wil::unique_com_class_object_cookie s_cookie;
+    static wil::srwlock s_activatorInfoLock;
```

Comment:
These are static global variables. Doesn't this mean if 2 .cpp #include this header they each get their own copy of the variables?

Are you intending this to be unique to a single global variable across the feature?

Reply:
It should be unique to just PushNotificationManager.cpp

Reply:
I don't see a reason for the statics to be on the header. They can be placed in the cpp, so no other headers can reference them.